library/alloc/src/vec/mod.rs

```diff
@@ -148,7 +148,7 @@ use self::spec_extend::SpecExtend;
 #[cfg(not(no_global_oom_handling))]
 mod spec_extend;
 
-/// A contiguous growable array type, written as `Vec<T>` and pronounced 'vector'.
+/// A contiguous growable array type, written as `Vec<T>`, short for 'vector'.
```

Comment:
Well, it really is `Vec<T, A>`, pronounced Veck-tah. 😛

Reply:
Look, I moved away from Boston to avoid this sort of thing.